src/test/java/gov/di_ipv_fraud/pages/FraudPageObject.java

```diff
@@ -137,4 +156,59 @@ public void whoWeCheckDetailsWith(String page) {
             Driver.get().switchTo().window(newTb.get(0));
         }
     }
-}
+
+    public void userSearchByName(String username) {
+        assertURLContains("credential-issuer?cri=fraud-cri");
+        usersearchField.sendKeys(username);
+        usersearchButton.click();
+    }
+
+    public void userSearchWithNoName() {
+        assertURLContains("credential-issuer?cri=fraud-cri");
+        usersearchButton.click();
+    }
+
+    public void goTofraudCRILink() {
+        fraudCRILink.click();
+    }
+
+    public void userNameInJsonResponse() throws JsonProcessingException {
+        String result = JSONPayload.getText();
+        System.out.println("result = " + result);
```

Comment:
Can we switch these system.out.println lines to use the logger like in the configuration service
    private static final Logger LOGGER = LogManager.getLogger();
It makes these much easier to read in an environment if we need to

Reply:
@gds-jbentham As requested, I have removed the System.out.println lines and have now used logger instead.